tests/framework/pod.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 const (
-	busyboxImage = "quay.io/quay/busybox:latest"
+	busyboxImage = "quay.io/prometheus/busybox:latest"
```

Comment:
@mhenriks wondering how much should we trust prometheus as the busybox source here. Should we use a kubevirt-owned image instead (vm-killer)?

Reply:
hosting ourself would be better but I suspect we can trust prometheus